examples/launch_templates_with_managed_node_groups/launchtemplate.tf

```diff
@@ -0,0 +1,88 @@
+data "template_file" "launch_template_userdata" {
```

Comment:
Is this necessary ? See my comment on the `userdata.sh.tpl` file.

Reply:
I copied from existing code in this module for the unmanaged node groups and there was no cloudinit used hehe. 
So you might also want to make same change there then ?

Reply:
Yep. As soon I add the feature to generate userdata template for MNG, I'll update your example accordingly.